modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -88,6 +88,21 @@ There are two levels of filters to help drill down on the logs to make visible:
 * _Containers_ filter on names of containers.
 * _Pods_ filter on the names of pods. This is useful for specifying application names followed by a wildcard (`%`).
 
+== Forward Logs to Anypoint Monitoring
+
+Depending on your Anypoint Platform subscription, you can forward log information to Anypoint Monitoring:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Runtime Fabrics* in the sidebar navigation.
+. Select the Runtime Fabric based on the name used during installation.
+. From the *Log Forwarding* tab, select *Forward application logs to Anypoint Monitoring* checkbox.
+. Click *Apply changes* to confirm the changes.
+
+[NOTE]
+====
+* The *Log Forwarding* tab is hidden if the Runtime Fabric version is earlier than version 1.4.35.
```

Comment:
`1.4.35` is not a public-facing version. Maybe `1.4.44`?

Reply:
Updated - thank you!